dev-docs/modules/userId.md

```diff
@@ -842,6 +842,42 @@ pbjs.setConfig({
 });
 {% endhighlight %}
 
+### Novatiq Snowflake ID
+
+Novatiq proprietary dynamic snowflake ID is a unique, non sequential and single use identifier for marketing activation. Only the snowflake ID moves beyond the firewall, keeping all customer data (publisher / brand and telco) private, secure and under control. Our in network solution matches verification requests to the telco network ID, safely and securely behind the telco firewall .
```

Comment:
there are a few grammatical errors here, but more importantly I don't know what the word "firewall" means in this context. Using the term  "only" followed by firewall w/o giving the community a better understanding of what you mean is confusing.